File: week_three/wednesday.rb
def mutation?(word, contains_letters)
  contains_letters.split("")
  count = 0
    if word.include?(contains_letters[count])
      count += 1
      return true
    else return false
    end
end

p mutation?("burly", "ruby")    #=> true
p mutation?("burly", "python")  #=> false

File: week_seven/wednesday.js
function mutation(stringOne, stringTwo) {
  var testArray = stringTwo.split("");
  var contains = false;
  for (var i = 0; i < testArray.length; i++) {
    if (!stringOne.includes(testArray[i])) {
      return false;
    }
  }
  return true;
}

console.log(mutation("burly", "ruby"));    // true
console.log(mutation("burly", "python"));  // false